XiEditor.xcodeproj/project.pbxproj

```diff
@@ -319,6 +319,7 @@
 				AE499DCD1C82BB2B002D68AF /* Products */,
 			);
 			sourceTree = "<group>";
+			usesTabs = 0;
```

Comment:
I think it was agreed some time ago that we'd use tabs in xi-mac, so I don't think this is necessary.

Reply:
Interesting.  It seems like all the code I've encountered uses spaces. The indentation issues below are because my editor keeps wanting to use tabs! :) 

I'll fix up the indentation to match the file and revert this change.

Reply:
Woops, after further checking I realized the default (and what xi-mac actually use) are spaces. I'm not sure why this showed up as a change, maybe newer XCodes demand that you have it explicit now or something.

Reply:
I've been using Xcode 10.2 Beta 4.  I suspect that it now makes that setting explicit and, when missing, defaults to tabs. We might want to consider turning that on explicitly as other folks start to use Xcode 10.2.

I've removed this line for now and will post this Q into Zulip for further discussion. :)